### ISSUE TITLE
Notification when someone joins or leaves the voice channel, that is announced in voice channel

### ISSUE THREAD

**Issue by Papadouka** [SwitchbladeBot/switchblade#831]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Papadouka** [comment]
Would make it easier to follow who is online.

**Comment by pedrofracassi** [comment]
I don't think it would be viable due to Lavalink limitations. @SwitchbladeBot/backend what do you think?

**Comment by Doges** [comment]
yea, lavalink does not support that, theres no text to speech for websocket or lavalink, and idk if we can do that

**Comment by Doges** [comment]
> yea, lavalink does not support that, theres no text to speech for websocket or lavalink, and idk if we can do that, but the ring thing, i think we can

but idk, discord has the own ring /shrug
